tests/xharness/Logging/ConsoleLog.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Xharness.Logging {
+	// A log that writes to standard output
+	public class ConsoleLog : Log {
+		StringBuilder captured = new StringBuilder ();
+
+		public ConsoleLog ()
+			: base (null)
```

Comment:
>null [](start = 11, length = 4)

I think the parent abstract class should not have the `Logs` field. It is not using it in any way in its implementation and it is obvious from this call that it is not needed there. Just in the implementations

Reply:
Agreed, but lets not touch that until we verify it with @rolfbjarne